js/runner_test.go

```diff
@@ -2006,3 +2006,101 @@ func TestComplicatedFileImportsForGRPC(t *testing.T) {
 		})
 	}
 }
+
+func TestExecutionStats(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name, script, expErr string
+	}{
+		{"vu_ok", `
```

Comment:
Not sure I understand the drawbacks, but I don't like using maps for test cases because of unordered test runs. I know it shouldn't matter and tests shouldn't rely on the order, but as long as they don't I prefer knowing the order they start in.

Reply:
> I don't like using maps for test cases because of unordered test runs.

I am not saying to use maps but use `{name: "vu_ok", script: "some javascript"}`

>I know it shouldn't matter and tests shouldn't rely on the order, but as long as they don't I prefer knowing the order they start in.

Also https://go-review.googlesource.com/c/go/+/310033/ and .. if it doesn't matter it's better to be random so at least it continues to not matter ;)

Reply:
Also `t.Parallel` in practice should mean that they are already somewhat randomized, the proposed `-shuffle` shuffles top-level tests only from the looks of it ;)

Reply:
> I am not saying to use maps but use `{name: "vu_ok", script: "some javascript"}`

Ah, I see what you mean now :) Sure, I'll change it.

Reply:
Resolved by 8b24539a.